packages/UI/src/components/organisms/NavDropdown/NavDropdown.tsx

```diff
@@ -46,12 +46,12 @@ const NavDropdown = ({ title, isOpen = false, links }: NavDropdownProps) => {
         <Options>
           {links.map(({ text, href }, i) => {
             return (
-              <>
+              <div key={i}>
```

Comment:
please don't add a new div, this might change the behavior of the component, do this instead to add key prop:
```
<React.Fragment key={i}>
....
</React.Fragment>
```

Reply:
good idea thank you !